contracts/src/periphery/cert/interfaces/IEigenDACertVerifierRouter.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifierBase} from "src/periphery/cert/interfaces/IEigenDACertVerifierBase.sol";
+
+interface IEigenDACertVerifierRouter is IEigenDACertVerifierBase {
```

Comment:
Add comment describing what the purpose of this router is: proxy that holds access to past implementations, and where new implementations can only be added in the future, etc.